Process.c

```diff
@@ -211,32 +293,41 @@ static inline void Process_writeCommand(const Process* this, int attr, int basea
    }
 }
 
-void Process_outputRate(RichString* str, char* buffer, size_t n, double rate, int coloring) {
+void Process_printRate(RichString* str, double rate, bool coloring) {
+   char buffer[16];
+
    int largeNumberColor = CRT_colors[LARGE_NUMBER];
    int processMegabytesColor = CRT_colors[PROCESS_MEGABYTES];
    int processColor = CRT_colors[PROCESS];
+   int shadowColor = CRT_colors[PROCESS_SHADOW];
 
    if (!coloring) {
       largeNumberColor = CRT_colors[PROCESS];
       processMegabytesColor = CRT_colors[PROCESS];
    }
 
    if (isnan(rate)) {
-      RichString_appendAscii(str, CRT_colors[PROCESS_SHADOW], "        N/A ");
+      RichString_appendAscii(str, shadowColor, "        N/A ");
+   } else if (rate < 0.005) {
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f B/s ", rate);
+      RichString_appendnAscii(str, shadowColor, buffer, len);
    } else if (rate < ONE_K) {
-      int len = snprintf(buffer, n, "%7.2f B/s ", rate);
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f B/s ", rate);
       RichString_appendnAscii(str, processColor, buffer, len);
    } else if (rate < ONE_M) {
-      int len = snprintf(buffer, n, "%7.2f K/s ", rate / ONE_K);
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f K/s ", rate / ONE_K);
       RichString_appendnAscii(str, processColor, buffer, len);
    } else if (rate < ONE_G) {
-      int len = snprintf(buffer, n, "%7.2f M/s ", rate / ONE_M);
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f M/s ", rate / ONE_M);
       RichString_appendnAscii(str, processMegabytesColor, buffer, len);
    } else if (rate < ONE_T) {
-      int len = snprintf(buffer, n, "%7.2f G/s ", rate / ONE_G);
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f G/s ", rate / ONE_G);
+      RichString_appendnAscii(str, largeNumberColor, buffer, len);
+   } else if (rate < ONE_P) {
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f T/s ", rate / ONE_T);
       RichString_appendnAscii(str, largeNumberColor, buffer, len);
    } else {
-      int len = snprintf(buffer, n, "%7.2f T/s ", rate / ONE_T);
+      int len = snprintf(buffer, sizeof(buffer), "%7.2f P/s ", rate / ONE_P);
```

Comment:
Actually it'd have to be `KB/s`, `MB/s`, `GB/s`, `TB/s`, and `PB/s`.
And as we're actually using binary prefixes that'd be even `KiB/s`, `MiB/s`, `GiB/s`, `TiB/s`, and `PiB/s`.

/PedanticMode

Reply:
Do we want to spend two columns for a bit more readability?

Reply:
No.

*importantly points to `/PedanticMode`*